kernel/dev/serial/rs232.cpp

```diff
@@ -15,10 +15,13 @@
 #include <arch/arch.hpp>
 #include <dev/serial/rs232.hpp>
 #include <mem/heap.hpp>
+#include <lib/mutex.hpp>
 
 static px_ring_buff_t* read_buffer = NULL;
 uint8_t rs_232_line_index;
 uint16_t rs_232_port_base;
+px_mutex_t mutex_data;
```

Comment:
I think we need one mutex for both data and initialization.

Reply:
Fixed in 285ec66